test/react-native/features/fixtures/app/scenario_js/app/scenarios/HandledOverrideJsErrorScenario.js

```diff
@@ -0,0 +1,12 @@
+import Scenario from './Scenario'
+import Bugsnag from '@bugsnag/react-native'
+
+export class HandledOverrideJsErrorScenario extends Scenario {
+  async run () {
+    Bugsnag.startSession()
+    await this.timeout(750)
```

Comment:
Are these timeouts just to ensure the event is always the second request received by the mock server?

Reply:
Yep, that's the case